CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## v1.18.6 (2019-03-19)
+
+### Other changes
+
+ * enable new release process
+ * Migrate content from workflow and pytorch readmes into sphinx project
+ * Propagate Tags from estimator to model, endpoint, and endpoint config
```

Comment:
it feels a little weird to start applying this to commits that were made before the commit message prefixes were introduced

Reply:
fixed